recipe/symfony.php

```diff
@@ -22,17 +26,29 @@
 
 // Assets
 set('assets', ['web/css', 'web/images', 'web/js']);
-// Default true - BC for Symfony < 3.0
-set('dump_assets', true);
+
+// Requires non symfony-core package `kriswallsmith/assetic` to be installed
+set('dump_assets', false);
```

Comment:
Well, may be fix it right in 3.x. Need to investigate a little. @oanhnn your thoughts?